gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -402,9 +404,11 @@ gst_tensor_src_iio_set_channel_type (GstTensorSrcIIOChannelProperties * prop,
   gchar endianchar = '\0', signchar = '\0';
   gint arguments_filled;
   gboolean ret = TRUE;
+  guint storage_bits;
   arguments_filled =
       sscanf (contents, "%ce:%c%u/%u>>%u", &endianchar, &signchar,
-      &prop->mask_bits, &prop->storage_bits, &prop->shift);
+      &prop->mask_bits, &storage_bits, &prop->shift);
+  prop->storage_bytes = storage_bits >> 3;
```

Comment:
What if ```(stroage_bits & 0x7 > 0) == TRUE```? Don't we need to round-up?
(e.g., ``` ((stroage_bits - 1) >> 3) + 1 if storage_bits > 0```)?

Reply:
Yes, I agree. 
Also, need to store storage_bits along with storage bytes to handle signed data when `(storage_bits & 0x7 > 0)` .